viewer/src/public/LocalHostRevealManager.ts

```diff
@@ -4,89 +4,24 @@
 
 import { RevealManagerBase } from './RevealManagerBase';
 
-import { CadSectorParser } from '@/datamodels/cad/sector/CadSectorParser';
-import { MaterialManager } from '@/datamodels/cad/MaterialManager';
-import { SimpleAndDetailedToSector3D } from '@/datamodels/cad/sector/SimpleAndDetailedToSector3D';
-import { CadModelMetadataRepository } from '@/datamodels/cad/CadModelMetadataRepository';
-import { DefaultCadTransformation } from '@/datamodels/cad/DefaultCadTransformation';
-import { CadMetadataParser } from '@/datamodels/cad/parsers/CadMetadataParser';
-import { CadModelFactory } from '@/datamodels/cad/CadModelFactory';
-import { ByVisibilityGpuSectorCuller, PotreeGroupWrapper, PotreeNodeWrapper } from '@/internal';
-import { CachedRepository } from '@/datamodels/cad/sector/CachedRepository';
-import { CadModelUpdateHandler } from '@/datamodels/cad/CadModelUpdateHandler';
-import { CadManager } from '@/datamodels/cad/CadManager';
-import { LocalUrlClient as LocalHostClient } from '@/utilities/networking/LocalUrlClient';
+import { PotreeGroupWrapper, PotreeNodeWrapper } from '@/internal';
+import { LocalModelDataClient as LocalHostClient } from '@/utilities/networking/LocalModelDataClient';
 import { CadNode, NodeAppearanceProvider } from '@/datamodels/cad';
-import { PointCloudMetadataRepository } from '@/datamodels/pointcloud/PointCloudMetadataRepository';
-import { PointCloudFactory } from '@/datamodels/pointcloud/PointCloudFactory';
-import { PointCloudManager } from '@/datamodels/pointcloud/PointCloudManager';
-import { DefaultPointCloudTransformation } from '@/datamodels/pointcloud/DefaultPointCloudTransformation';
 import { RevealOptions } from './types';
-import { Subscription, combineLatest } from 'rxjs';
-import { distinctUntilChanged, map } from 'rxjs/operators';
-import { trackError } from '@/utilities/metrics';
+import { initMetrics } from '@/utilities/metrics';
+import { omit } from 'lodash';
```

Comment:
As far as I remember lodash always worked poorly with tree-shaking. Need to check if it enlarges our bundle on the whole lodash, or it's tree-shaken. But if it adds the whole lodash it's better to use `import omit from 'lodash/omit'`

Reply:
I tried to find a difference in result bundle for examples and there is no difference. Looks like cognite SDK already brings the whole lodash. But still, I think it's better not to import it like that and on lodash page it also said 

> Cherry-pick methods for smaller browserify/rollup/webpack bundles.

https://www.npmjs.com/package/lodash

Reply:
I sent a PR to our js sdk as well.

Reply:
Nice, great!